src/MagnumPlugins/GltfImporter/GltfImporter.cpp

```diff
@@ -3286,15 +3286,40 @@ Containers::Optional<MeshData> GltfImporter::doMesh(const UnsignedInt id, Unsign
         return {};
     }
 
+    /* Morph target attributes */
+    const Utility::JsonToken* gltfTargets = gltfPrimitive.find("targets"_s);
+    if(gltfTargets) {
+        if(!_d->gltf->parseArray(*gltfTargets)) {
+            Error{} << "Trade::GltfImporter::mesh(): invalid targets property in mesh" << _d->gltfMeshPrimitiveMap[id].first();
+            return {};
+        }
```

Comment:
Can you test these error cases as well? Those would go into the `mesh-invalid.gltf` file, with corresponding test cases added to the `MeshInvalidData` array in the test. 

It may be a bit annoying as changing the file would shift the reported line numbers, but hopefully not too tragic.

Reply:
Can you also add a case that verifies behavior with a morph target attribute having a different component count? It'd be a variant of the `"different vertex count for each accessor"` case, and ideally the error message that's printed would also mention the morph target ID in that case, for example as `mismatched vertex count for attribute TEXCOORD_0 in morph target 1, expected 3 but got 4`.

Reply:
Done, updating the line numbers wasn't too bad.